### DISCUSSION TITLE
Not able to delete a node that is added

### DISCUSSION THREAD

**Discussion by nags-git** [FlowTestAI/FlowTest#139] [General]
Not able to delete a node that is added already

**Comment by jsajal** [comment]
Hi @nags-git , can you tell which os and which version of FlowTestAI are you using or did you just use it as a node application?

also i am assuming you are clicking on the node, pressing delete and the node does not go away?
if possible please attach you flow file also and which node you are trying to delete

**Reply by jsajal** [reply]
just an fyi `start` node is non-deletable
